dts/arm/silabs/efr32fg13p.dtsi

```diff
@@ -0,0 +1,206 @@
+/*
+ * Copyright (c) 2020 Piotr Mienkowski
+ * Copyright (c) 2021 Yonatan Schachter
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <arm/armv7-m.dtsi>
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/i2c/i2c.h>
+#include "gpio_gecko.h"
+
```

Comment:
Since https://github.com/zephyrproject-rtos/zephyr/blob/master/dts/arm/silabs/efr32bg13p.dtsi is very similar to this file, it would make sense to extract the common content to a new file efr32xg13p.dtsi and include the new file in this file and from efr32bg13p.dtsi